addOns/automation/src/test/java/org/zaproxy/addon/automation/jobs/PolicyDefinitionUnitTest.java

```diff
@@ -279,4 +282,28 @@ void shouldWarnIfRulesNotList() {
         assertThat(
                 progress.getWarnings().get(0), is(equalTo("!automation.error.options.badlist!")));
     }
+
+    @ParameterizedTest
+    @ValueSource(
+            strings = {
+                "{}",
+                "",
+                "  defaultStrength: \n" + "  defaultThreshold: \n" + "  rules: ",
+                "rules: \n",
+                "defaultStrength:"
```

Comment:
I'd say in these cases we should use the defaults. The user is specifying something after all (even if not complete).

Reply:
Done